frontend/src/components/cluster/Charts.tsx

```diff
@@ -164,9 +165,14 @@ export function PodsStatusCircleChart(props: Pick<ResourceCircularChartProps, 'i
   const { items } = props;
   const { t } = useTranslation(['cluster', 'glossary']);
 
-  const podsReady = (items || []).filter(pod =>
-    ['Running', 'Succeeded'].includes(pod.status!.phase)
-  );
+  const podsReady = (items || []).filter((pod: Pod) => {
+    if (pod.status!.phase === 'Succeeded') {
+      return true;
+    }
+
+    const readyCondition = pod.status.conditions.find(condition => condition.type === 'Ready');
```

Comment:
I think we should change this to 
```const readyCondition = pod.status?.conditions.find(condition => condition.type === 'Ready');``` to not allow any unknown property invocation

Reply:
The pod is supposed to have the status, but I will change it with defensiveness in mind.